client/js/test/sum-util.test.js

```diff
@@ -0,0 +1,18 @@
+const { getSumFromArray } = require('../sum-util');
```

Comment:
Add a few more tests that test the functionality of your new feature. Especially test against non-happy paths and edge cases. For instance, “it correctly generates sum when the column includes non-number inputs”.